### PR TITLE
Angelo Brown Checkpoint 01

### PR DESCRIPTION
**Pull request by brown2206**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <link rel='stylesheet' href='styles.css'>
+    <title>Jesse's Animals</title>
```
Comment (amaseda) [inline index.html]: :+1:

File: index.html [hunk 2]
```diff
@@ -10,12 +11,12 @@
     <aside>
       <h2>A Side Bar</h2>
     </aside>
-    <div class='main'>
+    <main class='main'>
```
Comment (amaseda) [inline index.html]: :+1:

File: index.html [hunk 3]
```diff
@@ -10,12 +11,12 @@
...
       <h2>The Main Content</h2>
       <div class='left'>
 	This is the left side
 	<img src='https://jesse.sh/img/me.jpg'>
```
Comment (amaseda) [inline index.html]: Missing an `alt=` attribute here.

File: styles.css [hunk 4]
```diff
@@ -28,12 +28,23 @@ aside{
...
+}
 
 @media(max-width:600px){
   .main{
     background:red;
   }
 }
+h1, h2, h3, p {
+  margin: 0;
+}
```
Comment (amaseda) [inline styles.css]: This is close, although you wanted to make sure the direct children of certain elements have `margin: 0`. That would look something like this, using `header` as an example...

```css
header > * {
  margin: 0;
}
```

File: styles.css [hunk 5]
```diff
@@ -28,12 +28,23 @@ aside{
...
+  width:50%;
+  background:#ccc;
+  padding:1em;
+  border: 5px solid #aaa;
+  order: 2;
+}
 
 @media(max-width:600px){
   .main{
     background:red;
```
Comment (amaseda) [inline styles.css]: You need to use Flexbox's `order` property somewhere in here to change the order of elements when viewing this page on a mobile device.

**Comment by amaseda** [comment]
Don't see an answer to the git question. Let me know if I'm overlooking it.

**Comment by brown2206** [comment]
I was really struggling with this checkpoint.  I simply ran out of time.  I
will do better on the next go round.

**Pull request closed without merging**